samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client/Program.cs

```diff
@@ -0,0 +1,36 @@
+using dymaptic.GeoBlazor.Core;
+using dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client;
+using dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client.Models;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using System.Net.Http.Json;
+
+var builder = WebAssemblyHostBuilder.CreateDefault(args);
+
+var baseAddress = new Uri(builder.HostEnvironment.BaseAddress);
+using var bootstrapHttp = new HttpClient { BaseAddress = baseAddress };
+var tempConfig = new Dictionary<string, string?>();
+
+try
+{
+    var cfg = await bootstrapHttp.GetFromJsonAsync<ClientConfigResponse>("/api/config");
+    if (cfg is not null)
+    {
+        void Add(string k, string? v) { if (!string.IsNullOrWhiteSpace(v)) tempConfig[k] = v; }
```

Comment:
This isn't very idiomatic, creating a private one-line function to add properties to a dictionary. It doesn't save any space, and there's already a try/catch to catch null errors. In fact, this pattern means a null will _not_ throw, and will silently fail because we just aren't adding the values to the `tempConfig`.

```csharp
if (cfg is not null)
{
    tempConfig[cfg.GeoBlazorLicenseKey] = "GeoBlazor:RegistrationKey";
    tempConfig[cfg.ArcGISApiKey] = "ArcGISApiKey";
    tempConfig[cfg.ArcGISPortalUrl] = "ArcGISPortalUrl";
    tempConfig[cfg.ArcGISAppId] = "ArcGISAppId";
}
```

If you want the null errors to be clearer/more specific, you would do that by changing the `ClientConfigResponse` and the `JsonSerializerOptions` to get it to throw exceptions earlier.